<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1.1 (Python 3.8.10 on linux)" generated="20240111 18:29:58.566" rpa="false" schemaversion="4">
<suite id="s1" name="Alerts" source="/home/ridham/PycharmProjects/webAutomation/Exercise3/Tests/Alerts.robot">
<test id="s1-t1" name="Launch Browser" line="15">
<kw name="Open Browser" library="SeleniumLibrary">
<arg>https://demoqa.com/</arg>
<arg>chrome</arg>
<arg>options=add_experimental_option("detach", True)</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<msg timestamp="20240111 18:29:58.737" level="INFO">Opening browser 'chrome' to base url 'https://demoqa.com/'.</msg>
<msg timestamp="20240111 18:29:59.253" level="WARN">There was an error managing chrome (error sending request for url (https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json): error trying to connect: Connection reset by peer (os error 104)); using driver found in the cache</msg>
<status status="PASS" starttime="20240111 18:29:58.736" endtime="20240111 18:30:04.761"/>
</kw>
<kw name="Maximize Browser Window" library="SeleniumLibrary">
<doc>Maximizes current browser window.</doc>
<status status="PASS" starttime="20240111 18:30:04.762" endtime="20240111 18:30:04.865"/>
</kw>
<status status="PASS" starttime="20240111 18:29:58.736" endtime="20240111 18:30:04.866"/>
</test>
<test id="s1-t2" name="Open Alerts,Frames Section" line="19">
<kw name="Execute Javascript" library="SeleniumLibrary">
<arg>document.querySelectorAll('div.card.mt-4.top-card')[2].click();</arg>
<doc>Executes the given JavaScript code with possible arguments.</doc>
<msg timestamp="20240111 18:30:04.868" level="INFO">Executing JavaScript:
document.querySelectorAll('div.card.mt-4.top-card')[2].click();
Without any arguments.</msg>
<status status="PASS" starttime="20240111 18:30:04.867" endtime="20240111 18:30:04.939"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>3s</arg>
<doc>Pauses the test executed for the given time.</doc>
<msg timestamp="20240111 18:30:07.940" level="INFO">Slept 3 seconds</msg>
<status status="PASS" starttime="20240111 18:30:04.940" endtime="20240111 18:30:07.941"/>
</kw>
<kw name="Perform Action for Browser Window">
<kw name="Execute Javascript" library="SeleniumLibrary">
<arg>document.querySelectorAll('#item-0')[2].click();</arg>
<doc>Executes the given JavaScript code with possible arguments.</doc>
<msg timestamp="20240111 18:30:07.944" level="INFO">Executing JavaScript:
document.querySelectorAll('#item-0')[2].click();
Without any arguments.</msg>
<status status="PASS" starttime="20240111 18:30:07.943" endtime="20240111 18:30:07.992"/>
</kw>
<kw name="Click Button" library="SeleniumLibrary">
<arg>${NEWTAB}</arg>
<doc>Clicks the button identified by ``locator``.</doc>
<msg timestamp="20240111 18:30:07.994" level="INFO">Clicking button '//*[@id="tabButtonWrapper"]'.</msg>
<msg timestamp="20240111 18:30:08.249" level="INFO" html="true">&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td colspan="3"&gt;&lt;a href="selenium-screenshot-9.png"&gt;&lt;img src="selenium-screenshot-9.png" width="800px"&gt;&lt;/a&gt;</msg>
<msg timestamp="20240111 18:30:08.249" level="FAIL">Button with locator '//*[@id="tabButtonWrapper"]' not found.</msg>
<status status="FAIL" starttime="20240111 18:30:07.993" endtime="20240111 18:30:08.287"/>
</kw>
<status status="FAIL" starttime="20240111 18:30:07.941" endtime="20240111 18:30:08.287"/>
</kw>
<status status="FAIL" starttime="20240111 18:30:04.866" endtime="20240111 18:30:08.289">Button with locator '//*[@id="tabButtonWrapper"]' not found.</status>
</test>
<status status="FAIL" starttime="20240111 18:29:58.567" endtime="20240111 18:30:08.293"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="1" skip="0" id="s1" name="Alerts">Alerts</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20240111 18:29:59.253" level="WARN">There was an error managing chrome (error sending request for url (https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json): error trying to connect: Connection reset by peer (os error 104)); using driver found in the cache</msg>
</errors>
</robot>
